.vscode/settings.json

```diff
@@ -17,9 +17,10 @@
     "logs/**": true
   },
   "python.testing.pytestArgs": [
-    "tests"
+    "."
   ],
   "python.testing.unittestEnabled": false,
   "python.testing.pytestEnabled": true,
```

Comment:
This shouldn't be here.

Reply:
I did some experimentation and you can just include a `tests/` folder in `benchmarks/swe_bench/` and then run `pytest` directly from that folder (you don't need to create a Python project or anything, `pytest` will just look for a `tests/` subdirectory and run what is there). Let's do it this way, as I do like the idea of tests for benchmarks but I don't want them part of the main test suite for the package.